safesql.go

```diff
@@ -133,16 +138,37 @@ func main() {
 		fmt.Printf("Found %d potentially unsafe SQL statements:\n", len(bad))
 	}
 
+	potentialBadStatements := []token.Position{}
 	for _, ci := range bad {
-		pos := p.Fset.Position(ci.Pos())
-		fmt.Printf("- %s\n", pos)
+		potentialBadStatements = append(potentialBadStatements, p.Fset.Position(ci.Pos()))
+	}
+
+	issues, err := CheckIssues(potentialBadStatements)
+	if err != nil {
+		fmt.Printf("error checking for ignore comments: %v\n", err)
+		os.Exit(2)
 	}
+
 	if verbose {
 		fmt.Println("Please ensure that all SQL queries you use are compile-time constants.")
 		fmt.Println("You should always use parameterized queries or prepared statements")
 		fmt.Println("instead of building queries from strings.")
 	}
-	os.Exit(1)
+
+	hasNonIgnoredUnsafeStatement := false
+
+	for _, issue := range issues {
+		if issue.ignored {
+			fmt.Printf("- %s is potentially unsafe but ignored by comment\n", issue.statement)
```

Comment:
If you want, you can use the `%q` format string, which will quote the string automatically.  I find it to be helpful when including strings in an output message.

Reply:
I am hesitant to change this as I am not sure if anyone is parsing this statement

Reply:
Fair enough.  I think in a future release (now that we'll have releases!), it would be nice to modify this to use the standard linter-style output format that you get with Go's `analysis` package.  That way, it'll integrate nicely with other Go tooling.